app/models/cart.rb

```diff
@@ -29,6 +29,25 @@ def remove_items(params)
     contents.delete_if { |item_id, _quantity| item_id == params[:id] }
   end
 
+  def complete_cart
+    {
+      items: items,
+      total_price: total_price
+    }
+  end
+
+  def items
+    item_ids = contents.keys
+    Item.find(item_ids)
+  end
+
+  def total_price
+    contents.map do |item_id, quantity|
+      Item.find(item_id.to_i).price * quantity
+    end
+    .sum
```

Comment:
Place the . on the previous line, together with the method call receiver.<br>Use 2 (not 0) spaces for indenting an expression spanning multiple lines.